arches/app/media/js/views/graph-designer.js

```diff
@@ -34,7 +34,9 @@ define([
             viewModel.graphSettingsVisible = ko.observable(false);
             viewModel.graph = koMapping.fromJS(data['graph']);
             viewModel.ontologies = ko.observable(data['ontologies']);
-            viewModel.ontologyClasses = ko.observable(data['ontologyClasses']);
+            viewModel.ontologyClasses = ko.pureComputed(function(){
```

Comment:
I have Linked Art loaded, but this change doesn't seem to affect the available options.

Reply:
This only "works" if you have more than one ontology loaded.  Eg, both CIDOC CRM and Linked.Art  
I used SHPO which has both of those ontologies.

I just tested again and this works. 

**Searching for String with CIDOC CRM**
<img width="555" alt="Screenshot 2024-04-05 at 9 41 52 AM" src="https://github.com/archesproject/arches/assets/823773/af31b773-6fe0-42fd-ae0e-e56f29fe2741">

**Searching for String with Linked.Art**
<img width="596" alt="Screenshot 2024-04-05 at 9 41 36 AM" src="https://github.com/archesproject/arches/assets/823773/9ece5c57-b65d-44a7-8164-39634f4f2770">